src/py/flwr/client/supernode/app.py

```diff
@@ -66,12 +66,23 @@ def run_supernode() -> None:
             "Ignoring `--flwr-dir`.",
         )
 
+    # Warn deprecated argument
+    if args.app is not None:
```

Comment:
@panh99 we need to update the docstring of this arg to indicate that it's no longer supported